localization/english/wc_other_titles_l_english.yml

```
@@ -1,173 +1,674 @@
+ d_karazhan_adj:0 "Karazhanian"
  
- e_burning_legion:0 "Burning Legion"
- e_burning_legion_adj:0 "Legion"
- d_frostmane:0 "Frostmane"
- d_riverpaw:0 "Riverpaw"
- d_redridge:0 "Redridge"
- d_drysnout:0 "Drysnout"
- d_mosshide:0 "Mosshide"
- d_bluegill:0 "Bluegill"
- d_wildfin:0 "Wild Fin"
- d_marsh:0 "Marshfin"
- d_marsh_adj:0 "Marshfin"
- d_murkgill:0 "Murkgill"
- d_drywhisker:0 "Drywhisker"
- d_tunnel_rat:0 "Tunnel Rat"
- d_witherbark:0 "Witherbark"
- d_vilebranch:0 "Vilebranch"
- d_greengill:0 "Greengill"
- d_vile_fin:0 "Vile Fin"
- d_mudsnout:0 "Mudsnout"
- d_blackpaw:0 "Blackpaw"
  d_caer_darrow:0 "Caer Darrow"
- GNOLL_BAND:0 "Gnoll Band"
- k_orc_blackrock:0 "Blackrock"
- d_shadow_council:0 "Shadow Council"
- d_twilights_hammer:0 "Twilight's Hammer"
- d_frostwolf:0 "Frostwolf"
- d_dragonmaw:0 "Dragonmaw"
- d_burning_blade:0 "Burning Blade"
- d_order_of_the_silver_hand:0 "Order of the Silver Hand"
- d_revantusk:0 "Revantusk"
- d_guardian_of_tirisfal:0 "Guardian of Tirisfal"
+ d_caer_darrow_adj:0 "Caer Darrowian"
+
+ ### Trolls ###
+ e_empire_of_zul:0 "Empire of Zul"
+ e_empire_of_zul_adj:0 "Zulian"
+ e_empire_of_zul_article:0 "the "
+
+ k_atalai:0 "Atal'ai tribe"
+ k_atalai_adj:0 "Atal'ai"
+ k_atalai_article:0 "the "
+
+ d_bloodscalp:0 "Bloodscalp tribe"
+ d_bloodscalp_adj:0 "Bloodscalp"
+ d_bloodscalp_article:0 "the "
+
+ d_skullsplitter:0 "Skullsplitter tribe"
+ d_skullsplitter_adj:0 "Skullsplitter"
+ d_skullsplitter_article:0 "the "
+
+ d_frostmane:0 "Frostmane tribe"
+ d_frostmane_adj:0 "Frostmane"
+ d_frostmane_article:0 "the "
+
+ d_darkspear:0 "Darkspear tribe"
+ d_darkspear_adj:0 "Darkspear"
+ d_darkspear_article:0 "the "
+
+ d_witherbark:0 "Witherbark tribe"
+ d_witherbark_adj:0 "Witherbark"
+ d_witherbark_article:0 "the "
+
+ d_vilebranch:0 "Vilebranch tribe"
+ d_vilebranch_adj:0 "Vilebranch"
+ d_vilebranch_article:0 "the "
+
+ d_revantusk:0 "Revantusk tribe"
+ d_revantusk_adj:0 "Revantusk"
+ d_revantusk_article:0 "the "
+
+ d_mossflayer:0 "Mossflayer tribe"
+ d_mossflayer_adj:0 "Mossflayer"
+ d_mossflayer_article:0 "the "
+
+ d_smoulderthorn:0 "Smolderthorn tribe"
+ d_smoulderthorn_adj:0 "Smolderthorn"
+ d_smoulderthorn_article:0 "the "
+
+ ### Dragons ###
+ e_green_dragonflight:0 "Green Dragonflight"
+ e_green_dragonflight_adj:0 "Verdant"
+ e_green_dragonflight_article:0 "the "
+
+ e_black_dragonflight:0 "Black Dragonflight"
+ e_black_dragonflight_adj:0 "Obsidian"
+ e_black_dragonflight_article:0 "the "
+
+ e_red_dragonflight:0 "Red Dragonflight"
+ e_red_dragonflight_adj:0 "Vermillion"
+ e_red_dragonflight_article:0 "the "
+
+ e_blue_dragonflight:0 "Blue Dragonflight"
+ e_blue_dragonflight_adj:0 "Azure"
+ e_blue_dragonflight_article:0 "the "
+
+ e_bronze_dragonflight:0 "Bronze Dragonflight"
+ e_bronze_dragonflight_adj:0 "Bronze"
+ e_bronze_dragonflight_article:0 "the "
+
+ ### Goblins ###
  e_trade_coalition:0 "Trade Coalition"
- d_blackwater_raiders:0 "Blackwater Raiders"
+ e_trade_coalition_adj:0 "Goblin"
```

You assume it's controlled by goblins.
```suggestion
 e_trade_coalition_adj:0 "Coalition"
```

---

done in ee5060e26